index.html

```diff
@@ -193,17 +193,29 @@ <h2><dfn>Performance Timeline</dfn></h2>
     <p>Each <a data-cite="WebIDL#es-environment">ECMAScript global
     environment</a> has:</p>
     <ul>
-      <li>a <dfn>performance observer task queued flag</dfn></li>
-      <li>a <dfn>list of <a>registered performance observer</a> objects</dfn>
-      that is initially empty</li>
-      <li>a <dfn>performance entry buffer</dfn> to store
-      <a>PerformanceEntry</a> objects that is initially empty
+      <li>A <dfn>performance observer task queued flag</dfn>.</li>
+      <li>A <dfn>list of <a>registered performance observer</a> objects</dfn>
+      that is initially empty.</li>
+      <li>A <dfn>performance entry buffer</dfn> to store
+      <a>PerformanceEntry</a> objects that is initially empty.
+      </li>
+      <li>
+        A <dfn>frozen supported entry types</dfn> representing the cached
```

Comment:
Let's remove the word "cached" here since that's not really a concept in WebIDL.

Reply:
Done